contracts/staking/StakingExtension.sol

```diff
@@ -26,6 +26,8 @@ contract StakingExtension is StakingV4Storage, GraphUpgradeable, IStakingExtensi
 
     /// @dev 100% in parts per million
     uint32 private constant MAX_PPM = 1000000;
+    /// @dev Minimum delegation for the first delegator of an indexer
+    uint256 private constant MINIMUM_DELEGATION = 1e18;
```

Comment:
Too bad we need to define `MINIMUM_DELEGATION` in two different files, is it too bad if we do it in `IStakingData.sol`? Only way that both files could read it from the same place.

Reply:
nvm it's an interface not a contract, so we can't have variables in there.